bench.mjs

```diff
@@ -3,7 +3,7 @@
  * Usage:
  * 	> npm run bench
  */
-import * as D from './lib/index.mjs';
+import * as D from './dist/esm/index.js';
```

Comment:
This now requires one to run `npm run build` first. I couldn't see a convenient way to have this file either be or directly import TypeScript.